bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -77,8 +77,11 @@ impl Room {
         self.inner.room_id().to_string()
     }
 
-    pub fn name(&self) -> Option<String> {
-        self.inner.name()
+    /// Returns the room's name from the state event if available, otherwise
+    /// compute a room name based on the room's nature (DM or not) and number of
+    /// members.
+    pub fn name(&self) -> Result<String, ClientError> {
+        Ok(RUNTIME.block_on(self.inner.computed_display_name())?.to_string())
```

Comment:
Would it be possible to have both `name` and `display_name` API?

There is a case when we want to use the name from the state event: editing the room name.

Currently we use the computed name, which is not correct in this case:

<img width="790" alt="image" src="https://github.com/matrix-org/matrix-rust-sdk/assets/3940906/a4f6a2dc-b4cb-4334-b2a5-ac7a2527f559">

So from the POV of a developer who use the SDK with some MAtrix knowledge background, having two API like:
- `name(): String?` the raw name, i.e. the name from the `m.room.name` state event, can be null
- `displayName(): String`, equals to the raw name if available, or a fallback using heroes
would be awesome.

Reply:
Yep, I've added `raw_name()` in the latest iteration. Thanks!